umm-spec-lib/src/cmr/umm_spec/validation/platform.clj

```diff
@@ -19,7 +19,41 @@
                  (vu/unique-by-name-validator :ShortName)]
    :Characteristics (vu/unique-by-name-validator :Name)})
 
+(def ^:private unique-fields
+  "Defines the fields to use that combined create unique platform names.
+  GCMD's keyword management system (KMS) defines aircraft, boats, ballons, etc.
+  with the model number. So the short name is not specific enough to identify
+  specific platform instances. Therefore we are adding Characteristic names and values
+  so that specific aircraft tail numbers, or boat IDS, etc. can be specified with the
+  short name."
+  #{:ShortName :Characteristics :Name :Value})
+
+(defn- get-platform-identifiers
```

Comment:
I actually asked Erich to leave it here. The other namespace was for generic validators that anyone could use. This validation is specific to platforms, so I think this is the right place.